src/backend/joanie/core/utils/contract.py

```diff
@@ -149,3 +151,21 @@ def generate_zip_archive(pdf_bytes_list: list, user_uuid: str, zip_uuid=None) ->
     storage.save(name=zip_archive_name, content=zip_buffer)
 
     return zip_archive_name
+
+
+def organization_has_owner(order_uuid=None) -> bool:
+    """
+    Returns a True whether we can find at least one organization owner
+    with the appropriate access rights, otherwise we return False.
+    """
+    try:
+        order = Order.objects.select_related("organization").get(pk=order_uuid)
+    except Order.DoesNotExist as error:
+        error_message = f"Order {order_uuid} does not exist"
+        logger.warning(error_message)
+        raise ValidationError(error_message) from error
```

Comment:
IMO there is no need to raise a validation error here. You have 2 solutions.

Don't use try/catch and catch the exception in the caller.
return False of the order does not exists.

Reply:
If I pass the object Order as a parameter for the method, i will not need to verify if the object exists.